Tools/ardupilotwaf/gbenchmark.py

```diff
@@ -10,11 +10,15 @@
 from waflib.TaskGen import feature, before_method, after_method
 from waflib.Errors import WafError
 
+import boards
+
 def configure(cfg):
     env = cfg.env
     env.HAS_GBENCHMARK = False
 
-    if env.TOOLCHAIN != 'native':
+    board = boards.get_board(cfg.env.BOARD)
+
+    if board.toolchain != 'native':
```

Comment:
See this: https://github.com/diydrones/ardupilot/pull/3765/commits/5e85b58f1c534dba7f440803be5e85b8ce878da4#r56679272